### ISSUE TITLE
Custom curve values changing after reloading the model in 1.0.3.

### ISSUE THREAD

**Issue by df7320** [FrSkyRC/ETHOS-Feedback-Community#117]
Here is a test plane I just created after installing 1.0.3 with a simple custom curve.  Notice the values and the placement of the dots.

![20210424_010750](https://user-images.githubusercontent.com/81670152/115948677-7124f400-a49d-11eb-98b2-bf7900b964e6.jpg)

Now exit out to the main menu and either power down the radio or load another model, then load the first model again.  Now go back to the Curve menu and notice the results.  Models created under 1.0.2 come up with the same type of random numbers after upgrading to 1.0.3.

![20210424_011228](https://user-images.githubusercontent.com/81670152/115948741-f7d9d100-a49d-11eb-9e7e-08f030d00ebd.jpg)

**Comment by OldGazer** [comment]
Reproduced on an X20.  
![Throttle Curve](https://user-images.githubusercontent.com/24305224/115957865-ee1e9080-a4d2-11eb-96b1-a52f36a893e0.jpg)

**Comment by centertrim** [comment]
Confirmed on X20 v1.0.3

Tested by adding curves directly in the curves menu.  Added custom curve with 2 points, changed endpoint y values to -100% & +100%.  I then cloned this curve three times, added points to each clone, and adjusted values on each curve.  Exited curves menu, switched to a different model, then switched back to the test model.  All four curves now show impossibly high Y values similar to OldGazer's image.

Oddly I did have an existing 13 point curve on my test model where 11 of the points changed to values around 4096%, but the x=-50%/y=-50% and x=-34%/y=-34% values were not affected.  I then changed to a different model, and then switched back to the test model.  The curve values changed to completely different values, but now the 0%, 16%, and 33% values were correct again.  

I tested an output associated with a simple 2-point linear curve that showed the impossible values in the curves menu.  The output still showed a linear response to the stick, so my thought is that the values may be stored correctly internally in the model file but the displayed values in the curves menu are incorrect.
